src/main/java/io/confluent/idesidecar/restapi/models/graph/FakeLocalFetcher.java

```diff
@@ -34,6 +34,6 @@ public Uni<ConfluentLocalKafkaCluster> getKafkaCluster(String connectionId) {
 
   @Override
   public Uni<LocalSchemaRegistry> getSchemaRegistry(String connectionId) {
-    return Uni.createFrom().item(new LocalSchemaRegistry(connectionId));
+    return Uni.createFrom().item(new LocalSchemaRegistry("http://localhost:8081", connectionId));
```

Comment:
Most of the other types are instantiated in code that doesn't know the connection (most are parsed from JSON responses with external services), and then we use the `withConnectionId(id)` to add that later. So there's no much precedence for passing the connection ID in right away. 

Having said that, would this be a little more clear if the connection ID were first, since most of the methods in the *Fetcher classes take the connection as the first parameter?

Reply:
I agree, connection as the first parameter is better. 
Done.